css/amp.css

```diff
@@ -502,10 +502,10 @@ i-amphtml-scroll-container.amp-active, i-amp-scroll-container.amp-active {
 
 /**
  * The default amp-hidden changes visibility, which takes up space. In the case
- * where amp-list becomes a container via the auto-resize attribute, the loader
- * should be fully hidden and not take up space.
+ * where amp-list becomes a container via the resizable-children attribute, the
+ * loader should be fully hidden and not take up space.
  */
-amp-list[auto-resize] > .i-amphtml-loading-container.amp-hidden {
+amp-list[resizable-children] > .i-amphtml-loading-container.amp-hidden {
```

Comment:
new name to be vetted later in design review meeting, since under an experiment flag, we can change it. For now went with one of the candidate name.